package.json

```diff
@@ -50,8 +50,8 @@
     "jsonpath": "^1.1.1",
     "mime-types": "^2.1.35",
     "node-fetch": "^2.6.1",
-    "oas": "^18.4.4",
-    "oas-normalize": "^7.0.0",
+    "oas": "^19.0.1",
```

Comment:
Nothing new there that's going to be used within the library, I just released a v19 last week so might as well update it here.

Reply:
Do we even use `oas` anywhere? Nothing comes up during my quick search

Reply:
Ignore me, it's used in the reducer.

Reply:
We load it in for the reducer command as that code is in `oas`.